### ISSUE TITLE
Paleoclimate data

### ISSUE THREAD

**Issue by lhmarsden** [cf-convention/discuss#139]
How should paleoclimate data be represented using the CF conventions? Can existing standard names like these still be used?

- sea_surface_temperature
- sea_water_salinity

Or should new more specific standard names be introduced? Another option would be provide a long name of "years since XXXX" or "year before 2021". What would you recommend as the best practice?

Also, can parameters like the age of sediments be represented?

**Comment by zklaus** [comment]
You can absolutely use those standard names for paleo data. Time information is usually not encoded in the standard or long name, but rather in a separate time coordinate. Please have a look at [CF Conventions 1.9, Section 4.4](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#time-coordinate) and [CF Conventions 1.9, Section 7.4](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#climatological-statistics).

Hope that helps. Do come back with any further questions.

**Comment by lhmarsden** [comment]
Thanks @zklaus this is clear.

What about the age of something? Let's say for example I have the age of seafloor sediments at different depths, or perhaps different latitudes and longitudes. I guess the age then becomes a variable rather than a dimension.

**Comment by zklaus** [comment]
> What about the age of something? Let's say for example I have the age of seafloor sediments at different depths, or perhaps different latitudes and longitudes. I guess the age then becomes a variable rather than a dimension.

Yes, I think that is correct. You can have a look at the CMIP6 data request for some examples. If you put "age" into the variable search box [here](http://clipc-services.ceda.ac.uk/dreq/mipVars.html), you will find a couple of variables like `siage` for age of sea ice. If you look for "age" in the CF Standard Name Table, for example in [the html version](http://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html) or [the KWIC index](http://cfconventions.org/Data/cf-standard-names/78/build/kwic_index_for_cf_standard_names.html#age), you will find a few suitable standard names.

If you find there are some missing, it would be appropriate to discuss their addition.

**Comment by lhmarsden** [comment]
Thanks @zklaus. I work as a data manager as part of a large project (>200 researchers). We are compiling a list of standard_names that are potentially missing from what is currently available. I will be in contact when this is done :)

**Comment by roy-lowry** [comment]
Totally agree with @zklaus. When specifying a standard name for sediment age (presumably 14C, U/Th or similar), which I'm pretty sure doesn't exist, it's worth giving the units some consideration. The canonical unit for any standard name with the dimensionality of time is second. The actual unit of measure is stored in a parameter attribute in the data file, which needs to be valid in UDUNITS. This includes 'years' but doesn't include anything like BP (years before present) which is often used in radiodating. Consequently, the date reference semantics would need to be included in the standard name - for example sea_floor_sediment_age_before_present. Existing standard name 'sea_floor_sediment_grain_size' was used as the model for the syntax.  Hope this helps.

BTW - for the benefit of non-geologists present is defined as 1950 AD.
